tests/unit/test-class-wc-payments-order-service.php

```diff
@@ -697,14 +697,14 @@ public function test_mark_payment_capture_failed_null_intent_status() {
 	public function test_mark_payment_capture_expired_with_provider( $fraud_outcome, $expected_fraud_outcome, $expected_fraud_meta_box ) {
 		// Arrange: Create the intent, get the proper order status variations, and get the charge. Set the fraud outcome status.
 		$intent            = WC_Helper_Intention::create_intention( [ 'status' => Payment_Intent_Status::CANCELED ] ); // Stripe uses single 'l'.
-		$order_status      = Order_Status::CANCELLED; // WCPay uses double 'l'.
+		$order_status      = Order_Status::FAILED;
 		$wc_order_statuses = wc_get_order_statuses(); // WooCommerce uses single 'l' for US English.
```

Comment:
The comment seems to be not relevant anymore, or that's just me?

Reply:
Sorry this remained from earlier change. I have now removed it